### PR TITLE
Meteo data

### PR DESCRIPTION
**Pull request by dennycox**

### CODE REVIEW

File: notebook.ipynb [hunk 1]
```diff
@@ -92,37 +92,1248 @@
     "<h2><font color=slateblue> 4. Data Exploration & Visualization </font></h2>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "71ec4af5",
+   "metadata": {},
+   "source": [
+    "<h4>Explore the meteo data<h4>"
```
Comment (rawan-a-d) [inline notebook.ipynb]: Add a title like we did in our branch
`<h3><font color=steelblue>Meteo dateset </font></h3>`

File: notebook.ipynb [hunk 2]
```diff
@@ -92,37 +92,1248 @@
...
+       "104504  1.000000    3.222932  \n",
+       "225319  0.984353   86.969696  \n",
+       "112150  0.902503    0.000000  "
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```
Comment (rawan-a-d) [inline notebook.ipynb]: Add a sub title on top of each new code cell
e.g. `<h4><font color=mediumvioletred>Read CSV files </font></h4>`

File: notebook.ipynb [hunk 3]
```diff
@@ -92,37 +92,1248 @@
...
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(figsize=(25, 6))\n",
+    "scatter = ax.scatter(list(postalcode[\"ws\"]),\n",
+    "                list(postalcode[\"ssrd\"]),\n",
+    "                color=\"blue\")\n",
+    "\n",
+    "ax.set(title=\"Windspeed compared to the solar surface radiation downwards \\n\",\n",
```
Comment (rawan-a-d) [inline notebook.ipynb]: Add a semicolon at the end so no text appears above the graph.
Same goes for the rest of the graphs.

File: notebook.ipynb [hunk 4]
```diff
@@ -92,37 +92,1248 @@
...
+   "source": [
+    "group_by_day = df_meteo_air_pollution.groupby(pd.Grouper(freq='D', key='date')).mean()\n",
+    "ws_pm10_corr = group_by_day[\"ws\"].corr(group_by_day[\"pm2.5\"])\n",
+    "print(ws_pm10_corr)\n",
+    "plt.scatter(group_by_day[\"ws\"], group_by_day[\"pm10\"])\n",
+    "plt.plot(np.unique(group_by_day[\"ws\"]), np.poly1d(np.polyfit(group_by_day[\"ws\"], group_by_day[\"pm10\"], 1))\n",
+    "         (np.unique(group_by_day[\"ws\"])), color='red')\n",
+    "plt.title('Correlation between PM10 and windspeed')\n",
+    "plt.xlabel('Windspeed in m/s')\n",
+    "plt.ylabel('particulate matter <10um in ug/m3')"
```
Comment (rawan-a-d) [inline notebook.ipynb]: The ylabel "<10um in ug/m3" is not really clear
Can you maybe change it to "Inhalable particles with diameter of 10 micrometers or smaller"?

**Review by rawan-a-d — CHANGES_REQUESTED** [review]

File: notebook.ipynb (hunk 1 above)
Comment (Ranim1996) [reply notebook.ipynb]: I agree with Rawan's feedback on this pull request.

**Review by Ranim1996 — CHANGES_REQUESTED** [review]

File: notebook.ipynb (hunk 4 above)
Comment (jarnovdh15) [reply notebook.ipynb]: We have renamed the y label

File: notebook.ipynb (hunk 3 above)
Comment (jarnovdh15) [reply notebook.ipynb]: The output text is gone

File: notebook.ipynb [hunk 5]
```diff
@@ -92,37 +92,1306 @@
     "<h2><font color=slateblue> 4. Data Exploration & Visualization </font></h2>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "71ec4af5",
+   "metadata": {},
+   "source": [
+    "<h3><font color=steelblue>Explore the meteo data</font><h3>"
```
Comment (rawan-a-d) [inline notebook.ipynb]: Make this more generic
e.g Meteo dataset

File: notebook.ipynb [hunk 6]
```diff
@@ -92,37 +92,1306 @@
...
+    "postalcode = df_meteo[df_meteo['PC4'] == 5611]\n",
+    "postalcode.sample(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11db7a55",
+   "metadata": {},
+   "source": [
+    "<h3><font color=steelblue>Visualize meteo data</font><h3>"
```
Comment (rawan-a-d) [inline notebook.ipynb]: I don't think this heading is needed

File: notebook.ipynb [hunk 7]
```diff
@@ -92,37 +92,1306 @@
...
+    "      xlabel=\"Date and time\",\n",
+    "      ylabel=\"Windspeed in m/s\");\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b15b52f7",
+   "metadata": {},
+   "source": [
+    "<h3><font color=steelblue>Merge datasets of meteo and air pollution</font><h3>"
```
Comment (rawan-a-d) [inline notebook.ipynb]: This is not needed either

File: notebook.ipynb [hunk 8]
```diff
@@ -92,37 +92,1306 @@
...
+    "df_meteo_air_pollution = pd.merge(df_meteo, df_air_pollution[['PC4','date', 'pm10', 'pm2.5', 'no2',\t'no', 'so2']], on=['PC4', 'date'])\n",
+    "df_meteo_air_pollution.sample(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8942b5ce",
+   "metadata": {},
+   "source": [
+    "<h3><font color=steelblue>Look for corrolations between meteo data and air pollution data</font><h3>"
```
Comment (rawan-a-d) [inline notebook.ipynb]: This is not needed either

**Review by rawan-a-d — CHANGES_REQUESTED** [review]

**Review by Ranim1996 — APPROVED** [review]

**Review by rawan-a-d — APPROVED** [review]

**Review by Stefan415849 — APPROVED** [review]

**Pull request merged**